src/App.tsx

```diff
@@ -179,6 +189,9 @@ const Root = () => {
         {routes.map(route => (
           <RouteWithSubRoutes key={route.path} {...route} />
         ))}
+        <Suspense fallback={<div />}>
+          <AdditionalRoutes />
+        </Suspense>
```

Comment:
I don't think we should be placing `Suspense` as a children of `Switch`. Feels like an anti pattern. `Routes` are the only thing that should go here.

Reply:
> @aChanEP take a look at https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules for some instructions on how to do preloading and prefetching and what is the difference between the two.

Will do